contracts/modules/TransferManager/VolumeRestrictionTM.sol

```diff
@@ -0,0 +1,934 @@
+pragma solidity ^0.4.24;
+
+import "./ITransferManager.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../../libraries/BokkyPooBahsDateTimeLibrary.sol";
+
+contract VolumeRestrictionTM is ITransferManager {
+    
+    using SafeMath for uint256;
+
+    // permission definition
+    bytes32 public constant ADMIN = "ADMIN";
+
+    enum RestrictionType { Fixed, Variable }
+
+    struct VolumeRestriction {
+        uint256 allowedTokens;
+        uint256 allowedPercentageOfTokens;
```

Comment:
Yes could do, but when I keep like this to make the code more readable and understandable.

Reply:
I think merging them will be clearer as then it will be obvious that either it can be a percentage or hard amount. Not a combination of both. Currently, I had to search through the code to make sure that only one of these is used at a time and it's not something like pick lowest of either.

Reply:
then will do that change